src/main/resources/app.conf

```diff
@@ -59,7 +59,7 @@ app {
 
   # Instance attachments specific settings
   attachments {
-    volume = "file:///tmp"
+    volume = "file:///tmp/" # ALWAYS INCLUDE THE TRAILING SLASH
```

Comment:
we should normalize the path and handle with or without a trailing slash... this is easily forgotten and to be honest it should not be required.